Johnsona Procedure 3 PR - Troubleshooting Wi-Fi Router

alyssajohnsonunofficial opened the pull request:
# Procedure 3 - Troubleshooting a Wi-Fi Router

I began my third procedure, which is for medium-level users who need to troubleshoot issues with their Wi-Fi router. 


**Completed Work**
- Started on my third procedure: 'troubleshootwifirouter' 
- Procedure's goal: Successfully troubleshoot issues with Wi-Fi router. 
- I laid out the basic sections with headers, added some text and an image (I may look for a second one later), and made an empty numbered list for my steps to go into, as well as an empty materials section I'll fill in soon. 


# User Scenario - First-Time Troubleshooter

**Scenario**: "Pam and Gray are an elderly couple in their late 60s. They're experiencing issues with their Synology WRX650 (spotty connectivity, slow internet, etc). While they're familiar with the installation process and the different parts of the router, they aren't familiar with the troubleshooting process. They're looking for a helpful, easy-to-read guide that will help them identify the issue they need to troubleshoot right now, as well as provide them with several troubleshooting techniques they can use to resolve future issues." 


## Notable Review Requests

- Similar questions to before; does the introduction feel too short, is the format well-organized, and is the image of decent quality?
- This one is more sparse than the other 2 so far, just because it's a bit different in focus than the other two and requires more research. I'll add the content and images soon. Let me know if you notice any issues with what I've added so far!
- Should the language be adjusted in a certain way for this procedure? It's a very similar audience to the second procedure (medium-level users), so I didn't see a need to make the language any different, but I'm open to ideas. I just wasn't sure if the straightforward and professional tone fits here, or if it needs to be more casual and slow.

sezito2 commented on adepojua/p-website-check-color-contrast.md:
```diff
@@ -0,0 +1,44 @@
+7. **Save Your Design**
+   - Once everything passes, save your design. Congratulations, your website is now more accessible for all users!
+
+> ⚠ **Don’t skip this step!**  
```
I think this alert might be better placed either in the testing step or at the beginning of your procedure

alyssajohnsonunofficial committed:
Created files

alyssajohnsonunofficial committed:
Started Procedure 3 + PR

dmbrasw2 commented on johnsona/troubleshootwifirouter.md:
```diff
@@ -0,0 +1,17 @@
+- 
+- 
+
+![The vents of a Synology WRX650 Wi-Fi router](./../johnsona/assets/images/synology-wrx560-review7.jpg)
```
Good picture! And good alt text

gmsimon22 commented on johnsona/PR-troubleshoot-wifirouter.md:
```diff
@@ -0,0 +1,23 @@
+
+- Similar questions to before; does the introduction feel too short, is the format well-organized, and is the image of decent quality?
+- This one is more sparse than the other 2 so far, just because it's a bit different in focus than the other two and requires more research. I'll add the content and images soon. Let me know if you notice any issues with what I've added so far!
+- Should the language be adjusted in a certain way for this procedure? It's a very similar audience to the second procedure (medium-level users), so I didn't see a need to make the language any different, but I'm open to ideas. I just wasn't sure if the straightforward and professional tone fits here, or if it needs to be more casual and slow. 
```
since it's for medium-level users, I think you can have more straightforward language since they will likely know more and be focused on skimming to quickly perform the tasks

WBlan97 replied:
I second that!

gmsimon22 commented on johnsona/troubleshootwifirouter.md:
```diff
@@ -0,0 +1,17 @@
+# How to Troubleshoot a Synology WRX560 Wi-Fi Router
+
+This procedure will guide you through the process of troubleshooting your Synology WRX560 Wi-Fi router. 
```
This introduction is quite short. If you have more that you can add, like different reasons for troubleshooting (what issues might have popped up, etc.) I think it will fill it out a bit nicer

WBlan97 replied:
Yes, I think it can be beefed up. Maybe like "Have you ever been on an important work call and had your wifi randomly drop?" or something analogous.

gmsimon22 commented on johnsona/troubleshootwifirouter.md:
```diff
@@ -0,0 +1,17 @@
+
+![The vents of a Synology WRX650 Wi-Fi router](./../johnsona/assets/images/synology-wrx560-review7.jpg)
+
+## The Procedure 
```
With more subheadings on the tasks, I think the structure will be good.

WBlan97 replied:
Visual cue is nice here.

gmsimon22 reviewed (commented):
Here are my reviews. Looking forward to seeing the finished results!

WBlan97 commented:
The bones are all there! Excited to see what comes next.

alyssajohnsonunofficial committed:
Added my first image's codeline to the Images folder

I copied and pasted the code line for my first image into my Images folder for my own reference

alyssajohnsonunofficial committed:
Added a screenshot and copied/pasted its codeline into the screenshots folder

I added a screenshot to my page and copied/pasted the codeline for it into the screenshot folder for my own reference

alyssajohnsonunofficial committed:
Added a figure and caption

I added a figure number and a caption to my screenshot

alyssajohnsonunofficial committed:
Added another image and copied its code into the images folder

I added an image to the top of my page and then copied and pasted its codeline into the images folder for my own reference

alyssajohnsonunofficial committed:
Made a copy of the user scenario file and added my user scenario

I made a new copy of the user scenario file, added my title, and wrote out my user scenario for this procedure

alyssajohnsonunofficial committed:
Added my intro and converted one section into another

I added an intro and decided to replace my "What You'll Need" section with a "Terms to Know" section; the "what you'll need" part was implemented into my intro instead!

alyssajohnsonunofficial committed:
More intro changes

alyssajohnsonunofficial committed:
Changed the name of my instructions section

alyssajohnsonunofficial committed:
Added to my user scenario

alyssajohnsonunofficial committed:
Added finishing touches to intro and adjusted my figures

alyssajohnsonunofficial committed:
Added definition 1

alyssajohnsonunofficial committed:
Added my final 2 definitions

alyssajohnsonunofficial committed:
Added new headers for my methods

alyssajohnsonunofficial committed:
Added my steps and adjusted the headers

I added all my steps and adjusted my title headers accordingly.

alyssajohnsonunofficial committed:
Final touches on first method

alyssajohnsonunofficial committed:
Changed order of steps

alyssajohnsonunofficial committed:
Changed wording for alert

lingeringcode committed:
Feedback from Lindgren

lingeringcode committed:
addedum to feedback from Lindgren: note on staging/prereqs
